martha/martha_v3.js

```diff
@@ -87,6 +88,16 @@ const DG_COMPACT_THE_ANVIL = 'dg.anv0';
 const DG_COMPACT_CRDC = 'dg.4dfc';
 const DG_COMPACT_KIDS_FIRST = 'dg.f82a1a';
 
+// Google Cloud Functions gives us 60 seconds to respond. We'll use most of it to fetch what we can,
+// but not fail if we happen to time out when fetching a signed URL takes too long.
+let pencilsDownTime = 58;
+
+// For tests that need to probe the behavior near the limit of the Cloud Function timeout, we don't
+// want to have to wait 60 seconds.
+const overridePencilsDownTime = (seconds) => {
```

Comment:
🤢 , but it's better than making each test that needs to bump against the timeout (currently 2 test cases) take a minute to run.